### ISSUE TITLE
[bug] bud.entry import syntax breaks with Bud 6.6.x

### ISSUE THREAD

**Issue by retlehs** [roots/bud#1890]
### Describe the issue

Try to use `bud.entry` with `import` [as described in the docs](https://bud.js.org/docs/bud.entry). Examples:

1. via @joshuafredrickson (working example in < 6.6.0):

```javascript
app: ['@scripts/app', '@styles/app'],
'block-library': {
  import: '@styles/block-library',
  dependOn: 'app',
},
```

2.

```javascript
.entry({
  app: {
    import: ["@scripts/app", "@styles/app"],
  },
})
```

### Actual Behavior

```
$ bud build
[bud@6.6.1] [sage] [api] › ✘  error

 bud.entry
 Array must contain at least 2 element(s)
Expected object, received array
import item must be a string.

 usage docs: https://bud.js.org/docs/bud.entry

[bud@6.6.1] [sage] › ✘  error     exiting with code 1
```

**Comment by kellymears** [comment]
Can reproduce:

https://github.com/roots/bud/blob/1acf7c533eeab8b2a0280dda81646f2a05c7dd58/tests/reproductions/issue-1890/bud.config.js

**Comment by kellymears** [comment]
Fixed in 6.6.2 ✅
